script.js

```diff
@@ -47,4 +50,119 @@ buttonElement.addEventListener('click', function (event) {
     subElement.classList.toggle('active');
 });
 
+buttonRight.addEventListener('click', function (){
+    move('right');
+}); 
+buttonLeft.addEventListener('click', function (){
+    move('left');
+});
+
+let images = document.querySelectorAll('.carousel object');
+let slider = [];
+for (let i = 0; i < images.length; i++) {
+    slider[i] = images[i].data;
+    images[i].remove();
+}
+let boxes = document.querySelectorAll('.carousel__box');
+for (let i = 0; i < boxes.length; i++) {
+    boxes[i].remove();
+}
+
+let carouselTextObject = document.createElement('div');
+carouselTextObject = carouselText;
+
+let step = 0;
+let carousel =  Array.from(document.querySelectorAll('.carousel__box'));
 
+function carouselObject (position, left) {
+    let objectDiv = document.createElement('div');
+    let objectImg = document.createElement('object');
+    objectImg.data = slider[position];
+    objectImg.classList.add('carousel__object');
+    objectDiv.classList.add('carousel__box');
+    objectDiv.style.left = left;
+    if (position === 0) {
+        objectDiv.appendChild(carouselTextObject);
+    }
+    objectDiv.appendChild(objectImg);
+    document.querySelector('.carousel').appendChild(objectDiv);
+    carousel.push(objectDiv);
+}
+
+function draw() {
```

Comment:
This is actually a good direction to go for, but the naming could be better, would you be able to understand what the function does by just reading the name of this function?

Reply:
Ok, changed name to drawCarouselSlides